### PR TITLE
Added WT65-D

### PR DESCRIPTION
**Pull request by wilba**
Added WT65-D

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/wilba_tech/wt65_d/wt65_d.c [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    setPinOutput(F1);
+    keyboard_pre_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    if (led_update_user(led_state)) {
+        writePin(F1, led_state.caps_lock);
+    }
+    return true;
+}
```
Comment (zvecr) [inline keyboards/wilba_tech/wt65_d/wt65_d.c:16]:
```suggestion
```

File: keyboards/wilba_tech/wt65_d/config.h [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+// Keyboard Matrix Assignments
+#define MATRIX_ROW_PINS { E6, F0, F4, F6, F7 }
+#define MATRIX_COL_PINS { F5, D5, D3, D2, B7, B0, B3, C7, C6, B6, B5, B4, D7, D6, D4 }
+#define UNUSED_PINS
+#define DIODE_DIRECTION COL2ROW
+
+// Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed
+#define DEBOUNCE 5
+
```
Comment (zvecr) [inline keyboards/wilba_tech/wt65_d/config.h:26]:
```suggestion

#define LED_CAPS_LOCK_PIN F1

```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/wilba_tech/wt65_d/info.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "wilba.tech WT65-D", 
+    "url": "https://wilba.tech", 
+    "maintainer": "Wilba", 
+    "width": 16, 
+    "height": 5,
```
Comment (drashna) [inline keyboards/wilba_tech/wt65_d/info.json:6]:
```suggestion
```

File: keyboards/wilba_tech/wt65_d/rules.mk [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/wilba_tech/wt65_d/rules.mk:18]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/wilba_tech/wt65_d/rules.mk [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Comment (zvecr) [inline keyboards/wilba_tech/wt65_d/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
```

**Review by drashna — APPROVED** [review]

File: keyboards/wilba_tech/wt65_d/rules.mk [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt65_d/rules.mk:18]:
```suggestion
```

File: keyboards/wilba_tech/wt65_d/rules.mk [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt65_d/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**